syncer/db.go

```diff
@@ -456,3 +459,7 @@ func getBinaryLogs(db *sql.DB) ([]binlogSize, error) {
 	}
 	return files, nil
 }
+
+func (c *column) isGeneratedColumn() bool {
+	return strings.Contains(c.extra, "VIRTUAL GENERATED") || strings.Contains(c.extra, "STORED GENERATED")
```

Comment:
must `extra`  be uppercase?

Reply:
according to MySQL doc, `VIRTUAL GENERATED` and `STORED GENERATED` in `Extra` field are always uppercase

Reply:
`strings.Contains` is used because there may exist other data in `Extra`?

Reply:
I'm afraid some comment options are in it

Reply:
https://dev.mysql.com/doc/refman/5.7/en/show-columns.html
In MySQL doc, there are only four Extra fields: `auto_increment`, `on update`, `VIRTUAL GENERATED` or `VIRTUAL STORED`